x/axelarnet/module.go

```diff
@@ -267,19 +267,14 @@ func (am AppModule) OnRecvPacket(
 	packet channeltypes.Packet,
 	relayer sdk.AccAddress,
 ) ibcexported.Acknowledgement {
-	//TODO: split rate limit, axelar routed packets and gmp into separated middleware?
-
-	// IBC receives are rate limited on the Incoming direction (tokens coming in to Axelar hub).
-	if err := am.rateLimiter.RateLimitPacket(ctx, packet, nexus.Incoming, types.NewIBCPath(packet.GetDestPort(), packet.GetDestChannel())); err != nil {
-		return channeltypes.NewErrorAcknowledgement(err)
-	}
+	// TODO: split rate limit, axelar routed packets and gmp into separated middleware?
```

Comment:
this comment seems out of place now here

Reply:
updated